src/components/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { InputTitleText, InputText } from './styles';
+
+export default function TextInput() {
+  return (
+    <>
+      <InputTitleText>Label</InputTitleText>
+      <InputText $error={false} placeholder="Input Suggestion" />
+    </>
```

Comment:
Instead of making a test component containing these components, then inserting this into a test page, I think it would be cleaner to just put these components into the test page itself.

Unless we want to keep this component? Though it's probably not the best idea since it's just bundling two components together, adding more unnecessary complexity.

Reply:
sounds good! i'll just put the two components separately on the test page.